MIGRATION.md

```diff
@@ -0,0 +1,9 @@
+# Breaking changes
+
+- Properties `gatewayHost`, `gatewayPort`, `outboundPlatformUrl`, `proxyHost, `proxyPort`, `proxyUrl` and `pluginPort` of `BasePlugin` must be now accessed from the property `config`.
+- All gateway calls (like `fetchActivityAnalyzer`, `fetchRecommenderProperties`, …) of `BasePlugin` must be now accessed from the property `gatewaySdk`.
+- Mics API calls (`fetchDatamarts`, `fetchDatamartCompartments`) must be now accessed from the property `apiSdk`
+- `BasePlugin` constructor signature has changed. It’s now an object which can contains `config`, - `gatewaySdk`, `apiSdk` and the original `enableThrottling` property.
```

Comment:
Can you put this line at the top of the list -> it explain what is `config`for example, useful to understand what has changed for the "properties" fetching.